Add ls changes to 2201.6.0 release note (#6940)
ballerina-platform/ballerina-dev-website: downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md

### New features

#### Language Server

- Added completions for the `group by` clause.
- Added inlay hint support for function call expressions and method call expressions to provide information about parameters.

### Improvements

#### Language Server

- Removed service template initialization from the lightweight mode.

> - Improved completions in the client resource access action node context.

Reviewer: ```suggestion
- Improved the completion support and signature help for client resource access actions.
```

- Improved the main function completion item.
- improved completions in the named argument context.

> ### Bug fixes

Reviewer: Let's add a link to the LS bug fixes as well.

Eg: In the previous release note
![image](https://github.com/ballerina-platform/ballerina-dev-website/assets/35211477/e7e44a2f-5e58-4c8a-852b-b0b99770d407)

Author: There is a link already
